h/static/scripts/test/admin-users-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var AdminUsersController = require('../admin-users');
+
+describe('CreateGroupFormController', function () {
+  var element;
+  var template;
+
+  before(function () {
+    template = '<form id="js-users-delete-form">' +
+               '<input type="submit">';
+  });
+
+  beforeEach(function () {
+    element = document.createElement('div');
+    element.innerHTML = template;
```

Comment:
If you want to interact with an element in tests, you need to add it to the page, eg. using `document.body.appendChild()` and then remove it afterwards using eg:

```js
afterEach(function () {
   element.parentNode.removeChild(element);
})
```